kitchen.ec2.yml

```diff
@@ -67,6 +68,10 @@ platforms:
       <% end %>
     transport:
       username: ec2-user
+    attributes:
+      cluster:
+        ddb_table: <%= ENV['DDB_TABLE'] %>
```

Comment:
As far as I understand, this attribute is required for `head_node_base` test and not elsewhere. Can we add it to the specific test? Also, I suppose we don't use this table name but to resolve a variable in a template. It can literally be any value. And I would check the contents of `clusterstatusmgtd.conf` and verify there is a line:
```
dynamodb_table = WHATEVER_VALUE_YOU_SET
```